### DISCUSSION TITLE
Alternating background color for help

### DISCUSSION THREAD

**Discussion by g0mb4** [neomutt/neomutt#3929] [Development]
I think an alternating background color would help the readability of the help screen. I can give it a try. I would hide it behind an option, to keep the original behaviour.

**Comment by flatcap** [comment]
How about two `color`s:
```
color odd_line  fg bg
color even_line fg bg
```

If they're not set, they feature is 'off'.
It also allows using bold or underline for emphasis

**Reply by g0mb4** [reply]
I'll think an enable/disable option would be beneficial, allowing the theme creators to include the colors.

**Reply by flatcap** [reply]
yep, fair point

**Reply by g0mb4** [reply]
even fallback colors can be chosen handling the case when `odd_line` and `even_line` are not set, but the feature is enabled

**Reply by flatcap** [reply]
> fallback colors

That's a bit tricky if we don't know if the user's background colours (light/dark)

**Reply by g0mb4** [reply]
I mean `odd_line = indicator` and `even_line = index` or something like that.

**Reply by flatcap** [reply]
hmm... like vim's:
```vim
highlight link from to
```

Interesting

**Comment by flatcap** [comment]
Are there any other screens that would benefit from line highlighting?

**Comment by flatcap** [comment]
Would you like to work out the implementation for yourself?
Or would you like some hints?

**Reply by g0mb4** [reply]
I'll give it a try.

**Comment by g0mb4** [comment]
Sorry, I've solved it :)

**Reply by flatcap** [reply]
Yeah, I noticed :-)
Well done!  It's looking really good.

**Comment by roccoblues** [comment]
This has been done in https://github.com/neomutt/neomutt/commit/73d873e425b8f6e94b0ad78c64272fb15c5eaa33 and can be closed right?
